crates/primitives/src/db/components.rs

```diff
@@ -3,13 +3,16 @@ pub mod block_hash;
 pub mod state;
 
 pub use block_hash::{BlockHash, BlockHashRef};
+use hashbrown::HashMap;
```

Comment:
```suggestion
use crate::HashMap;
```
When `std` is enabled we are using std HashMap so in all places in code we include hashmap as `crate::HashMap`

Reply:
Addressed in https://github.com/bluealloy/revm/pull/1163/commits/9aca198e12bc82b7bc58fc3814db925f30535deb